docker-compose.yml

```diff
@@ -0,0 +1,85 @@
+services:
+  dittofs:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    container_name: dittofs-server
+    image: dittofs:latest
+    ports:
+      - "12049:12049/tcp"
+      - "2049:2049/udp"
+      - "9090:9090/tcp"
+    volumes:
+      - ~/.config/dittofs/config.yaml:/config/config.yaml:ro
+      - dittofs-metadata:/data/metadata
+      - dittofs-content:/data/content
+    restart: unless-stopped
+    cap_add:
+      - SYS_ADMIN
+    healthcheck:
+      test: ["CMD", "/app/dittofs", "help"]
+      interval: 30s
+      timeout: 10s
+      retries: 3
+      start_period: 10s
```

Comment:
This health check is fine for now, but it is definitely not secure for a production environment. We need to add a small health route to the Ditto server (we can also setup an HTTP server, because I am thinking to set it up anyway in the future to provide a management dashboard). Only then can we know the status of the running server. Help will never fail...

Reply:
we can delete it for now, it's not strictly necessary at this stage